src/app/component/InstallPrompt/InstallPrompt.component.js

```diff
@@ -51,14 +51,14 @@ export class InstallPrompt extends PureComponent {
     }
 
     renderPrompt() {
-        const { device } = this.props;
+        const { device, containerFunctions } = this.props;
```

Comment:
How is this related to compare?

Reply:
Not related. Just couldn't close this prompt when was testing on the mobile device. Fixed it.